### ISSUE TITLE
[question] How would I fetch all historic policy states?

### ISSUE THREAD

**Issue by jschirrmacher** [lacuna-tech/mds-core#1017]
I would like to visualize the historic development of a policy. In the policy API, I don't find a route to get these. Did I miss something or isn't there any such yet? Should we, maybe, add a GET route in the author-api?

**Comment by avatarneil** [comment]
I don't think that at the moment we have a way to derive historical policy state. Could definitely be worth adding!
